src/librbd/managed_lock/NotifyLockOwner.cc

```diff
@@ -0,0 +1,105 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Minor: I think this is a good generic helper for ```ObjectWatcher```, but not necessarily tied into the lock